lib/active_model/serializer/utils.rb

```diff
@@ -0,0 +1,24 @@
+module ActiveModel::Serializer::Utils
+  module_function
+
+  def include_string_to_hash(included)
+    included.delete(' ').split(',').inject({}) do |hash, path|
+      hash.deep_merge!(path.split('.').reverse_each.inject({}) { |a, e| { e.to_sym => a } })
+    end
+  end
+
+  def include_args_to_hash(included)
+    case included
+    when Symbol
+      { included => {} }
+    when Hash
+      Hash[included.map { |key, value| [key, include_args_to_hash(value)] }]
```

Comment:
Something like `included.reduce { |a, e| a.merge!(e.first => include_args_to_hash(e.second) }`? Idk, it felt more natural to iterate over the `key, value` pairs and combine the result as a Hash, but I wouldn't mind changing it.